doc/internal/hackers-guide.asciidoc

```diff
@@ -175,17 +195,31 @@ code, so it's only helpful for developers.
 
 === Use (and extend) the unit tests
 
-In the test subdirectory resides a simple unit test to check the most important
-functionality of the newsbeuter internals. These tests build upon the lemon
-test framework which comes bundled as test/lemon.h with newsbeuter. Run "make
-test" to build the tests, the result is a binary called "test" within the test
-subdirectory. Run it and see whether everything still works as expected. Run
-"make clean-test" to clean up after the tests.
+**Status of this section:** up-to-date (January 2021).
+
+C++ tests are in the _test_ subdirectory. They're using
+https://github.com/catchorg/Catch2/tree/devel/docs[the Catch2 framework]. Rust
+tests are split: unit tests are in `mod tests` sub-modules under
+_rust/libnewsboat/src_, while integration tests are under
+_rust/libnewsboat/tests_. Rust's test runner is multi-threaded, so we use
+integration testing when we want to do the checks in a separate process.
+
+In addition to `TMPDIR=/dev/shm make -j5 PROFILE=1 check` incantation that's
+already explained in the contributing guidelines, you should know about
```

Comment:
…about **the**…

Reply:
Thanks! Fixed in a33be00d8b7a60ab13cf37caf2bf41d08fe1b2a2.